src/helpers/primary-term-helper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Primary term helper.
+ *
+ * @package Yoast\WP\SEO\Helpers
+ */
+
+namespace Yoast\WP\SEO\Helpers;
+
+/**
+ * Class Primary_Term_Helper
+ */
+class Primary_Term_Helper {
+
+	/**
+	 * Returns all the taxonomies for which the primary term selection is enabled.
+	 *
+	 * @param int $post_id Default current post ID.
+	 *
+	 * @return array The taxonomies.
+	 */
+	public function get_primary_term_taxonomies( $post_id = null ) {
```

Comment:
Do we need this `= null` here? I think we should require an actual post ID and remove the double functionality here.